hu/hu_aap.cpp

```diff
@@ -492,6 +496,7 @@
       audioConfig->set_sample_rate(48000);
       audioConfig->set_bit_depth(16);
       audioConfig->set_channel_count(2);
+      inner->set_available_while_in_call(true);
```

Comment:
Same here

Reply:
I thought maybe if I set this to true it would override BTHF and take over sound handling but in testing it has no effect on anything